crates/emitter/src/aligner.rs

```diff
@@ -776,7 +776,10 @@ impl VerylWalker for Aligner {
         if let Some(ref x) = arg.function_declaration_opt0 {
             self.port_declaration(&x.port_declaration);
         }
-        self.minus_g_t(&arg.minus_g_t);
+        if let Some(ref x) = arg.function_declaration_opt1 {
+            self.minus_g_t(&x.minus_g_t);
+            self.scalar_type(&x.scalar_type);
```

Comment:
```suggestion
            // skip type align
            //self.scalar_type(&x.scalar_type);
```

Reply:
The `scalar_type` should be skipped.
This causes the following unexpected spaces.

```
function automatic logic        [ParamX-1:0] FuncA(
                         ^^^^^^^
        input  logic        [ParamX-1:0] a,
                     ^^^^^^^
```